cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1899,6 +1905,39 @@ pub(crate) fn define(
         e.enc_32_64_maybe_isap(instruction, template, None); // from SSE
     }
 
+    // SIMD register movement: store, load, spill, fill, regmove. All of these use variants of
+    // MOVUPS from SSE (TODO ideally this would use the aligned MOVAPS).
+    for ty in ValueType::all_lane_types().filter(allowed_simd_type) {
+        // Store
+        let bound_store = store.bind_vector_from_lane(ty, sse_vector_size).bind_any();
+        e.enc_32_64(bound_store.clone(), rec_fst.opcodes(vec![0x0f, 0x11]));
```

Comment:
For all of these, shouldn't we use enc_both instead, so this adds the rex-variant on x64, allowing uses of more xmm registers? (makes me wonder if enc_32_64_maybe_isap should do this too)

Reply:
Let me do some more research on this one; I think you're right from what I figured out in https://github.com/CraneStation/cranelift/pull/919 but I just want to make sure.

Reply:
I've convinced myself that this is safe to do. I am still planning to refactor `PerCpuModeEncodings` as a part of #955 so I can do it there (and remove all the unnecessary `enc_*` methods).

Reply:
Sounds good to me, thanks.